content/azure.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Microsoft Azure
+---
+
+# Microsoft Azure
+
+The `azure` CPI can be used with [Microsoft Azure](https://azure.microsoft.com/).
+
+ * Release: [cloudfoundry-incubator/bosh-azure-cpi-release](https://github.com/cloudfoundry-incubator/bosh-azure-cpi-release)
+ * Issues: [GitHub Issues](https://github.com/cloudfoundry-incubator/bosh-azure-cpi-release/issues)
+ * Slack: [cloudfoundry#bosh-azure-cpi](https://cloudfoundry.slack.com/messages/bosh-azure-cpi)
+
+
+## Concepts
+
+The following table maps BOSH concepts to their Azure-native equivalents.
+
+| BOSH              | Microsoft Azure |
+| ----------------- | --------------- |
+| Availability Zone | [Availability Zone](https://docs.microsoft.com/en-us/azure/availability-zones/az-overview) |
+| Virtual Machine   | [Virtual Machine](https://docs.microsoft.com/en-us/azure/virtual-machines/linux/sizes) |
+| Network Subnet    | [Virtual Network Subnet](https://docs.microsoft.com/en-us/azure/virtual-network/virtual-networks-overview) |
+| Virtual IP        | [Public IP](https://docs.microsoft.com/en-us/azure/virtual-network/virtual-network-ip-addresses-overview-arm#public-ip-addresses) |
+| Persistent Disk   | [Disk Storage](https://azure.microsoft.com/en-us/services/storage/disks/) and [Managed Disks](https://azure.microsoft.com/en-us/services/managed-disks/) |
+| Disk Snapshot     | [Managed Disk Snapshot](https://docs.microsoft.com/en-us/azure/virtual-machines/windows/managed-disks-overview#managed-disk-snapshots) |
+| Stemcell          |  |
+
+
+## Feature Support
+
+
+### Network
+
+| Network Type | Support |
+| ------------ | ------- |
+| Manual       | Multiple networks per instance |
+| Dynamic      | Multiple networks per instance |
+| VIP          | Single network per instance |
+
```

Comment:
Can we add an analogous sentence like we have for AWS? "The AWS CPI does not currently support multiple NICs being attached to a VM."

Reply:
Will do